store/docs/soap-calendar.txt

```diff
@@ -125,6 +125,7 @@
                 // Possible values are "subject", "location", "time" (start time, end time, or duration), and "recurrence".
             [draft="0|1"]       // set to 1 if invite has changes that haven't been sent to attendees; for organizer only
             [neverSent="0|1"]   // set to 1 if attendees have never been notified for this invite; for organizer only, used in SetAppointmentRequest only
+            [color="{color}"] [rgb="{rgb-color}"]
```

Comment:
I assume the JAXB classes corresponding to this need updating too.

Reply:
fixed some classes under zm-mailbox/soap; added color and rgb to InviteComponent and InstanceData